system_tests/espresso-e2e/.env

```diff
@@ -0,0 +1,28 @@
+# Environment for local demo network.
```

Comment:
We can probably use the docker compose files from the testnode for this to avoid having a copy of it, but we can do that in follow up PRs.

Reply:
https://github.com/EspressoSystems/nitro-espresso-integration/issues/34